scripts/rollup/bundles.js

```diff
@@ -360,7 +372,7 @@ const bundles = [
 
   /******* React Reconciler *******/
   {
-    bundleTypes: [NODE_DEV, NODE_PROD],
+    bundleTypes: [NODE_DEV, NODE_PROD], // todo - should this have a testing build?
```

Comment:
@acdlite thoughts? should there be a testing build for the reconciler package too?

Reply:
Doesn't really matter. The recommended way to build a custom renderer is to fork our build script.